main.go

```diff
@@ -62,10 +66,13 @@ func setupConfig() (config, error) {
 	fs.StringVar(&cfg.token, "token", "", "Axiom API token")
 	fs.StringVar(&cfg.url, "url", "", "Axiom API URL (optional)")
 	fs.StringVar(&cfg.orgID, "org-id", "", "Axiom organization ID")
+	fs.StringVar(&cfg.internalAuthToken, "internal-auth-token", "", "Axiom internal API auth token")
```

Comment:
What is this token and how is it different than token? I don't think we should expose anything that is internal...